app.py

```diff
@@ -95,5 +82,28 @@ def handle_standard_exception(ex):
     }), ex.code
 
 
+def register_url_rules(app):
+    app.add_url_rule("/api/restaurants", view_func=RestaurantEndpoint.as_view("restaurant_api"))
+    app.add_url_rule("/api/restaurants/<int:restaurant_id>",
+                     view_func=RestaurantItemEndpoint.as_view("restaurant_item_api"))
+    app.register_error_handler(CustomError, handle_exception)
```

Comment:
often error handlers are registered in another function like `register_error_handlers(app)`

Reply:
fixed